### ISSUE TITLE
Manual re-ordering of pages [sic] results in new time stamp for page.updated_on for all pages

### ISSUE THREAD

**Issue by dajare** [wolfcms/wolfcms#545]
**tl;dr** : manually re-ordering any page results in a new `updated_on` timestamp for _all_ pages -- is this desirable?

---

[This "issue" is related to #397 and #544 and is logged just to ensure we have identified all relevant complexities with the `position` discussion.]

---

This issue records an observation, which possibly leads to an enhancement request; it is **not** a "bug".

One of Wolf's pieces of core functionality is the simple, drag&drop page\* ordering in the backend "Pages" view. This is one of the distinctive features it inherited from Radiant, and factor that appeals to users (IMO).

> \* I will refer to "pages" rather than "nodes" because (a) I'm not sure I understand "nodes" ;) and (b) I'm taking a user-centric view based on present Wolf implementation.

As currently implemented, it has one side-effect that might surprise end users. When any page is manually re-ordered, at any point in a branch of pages, EVERY entry in the `page.updated_on` field is written with the same time stamp.

Where this becomes an "issue" is for those users who would be surprised to find a new `updated_on` date for some other page when that page itself has had no "updating". 

**Use case:** a charity keeps a "News" page where news items might be updated when new information becomes available. The `updated_on` date is used to reflect this on the front-end in the article's meta-data. However, if some page is manually re-ordered in some other branch of the site, all "News" pages will now show the re-ordering date in the "Updated on:" meta-data, when in fact no content has changed.

CONCLUDING QUESTIONS:
- is this behaviour desirable? (If yes, we maintain the _status quo_.)
- if not, is it simple to modify? (This would require that the `page.updated_on` field is NOT rewritten when doing a position change.)

p.s. This is an "information logging" issue and need not stay "open".
